stempy/reader.cpp

```diff
@@ -26,6 +26,24 @@ using std::vector;
 
 namespace stempy {
 
+Header::Header(uint32_t frameWidth, uint32_t frameHeight,
+               uint32_t imageNumInBlock,uint32_t blockNumInFile, uint32_t scanWidth, 
+               uint32_t scanHeight, uint32_t startPosition)
+{
+    this->frameWidth = frameWidth;
+    this->frameHeight=frameHeight;
+    this->imagesInBlock = imageNumInBlock;
+    this->scanHeight=scanHeight;
+    this->scanWidth=scanWidth;
+    //only suitable for big_stem_image.hdf5
+    uint32_t offset1 = startPosition/(imageNumInBlock*blockNumInFile);
+    uint32_t offset2 = startPosition%(imageNumInBlock*blockNumInFile);
```

Comment:
This is specific to a particular dataset?

Reply:
yeah, I only tested for big_stem_image, since the big_stem_image come from the smallscan data, and smallscan data has the header in dataset, here we set the header manually to avoid the modification for the original code.